platform/windows/SCsub

```diff
@@ -72,7 +87,9 @@ if env["d3d12"]:
         # Let's copy to both and let the user decide the distribution model.
         for v in ["#bin", arch_bin_dir]:
             env.Command(
-                v + "/" + dxc_dll, env["dxc_path"] + "/bin/" + arch_subdir + "/" + dxc_dll, Copy("$TARGET", "$SOURCE")
+                v + "/" + dxc_dll,
+                env["dxc_path"] + "/bin/" + dxc_arch_subdir + "/" + dxc_dll,
+                Copy("$TARGET", "$SOURCE"),
```

Comment:
Can this be formatted like this? I had wanted to do this, but the formatter would put everything on a single line, so I though that was needed for the static format checks.

Reply:
You can enforce this even on shorter lines by ensuring there is a trailing comma on the last line.